base/db/R/insert.format.vars.R

```diff
@@ -1,28 +1,67 @@
 #' Insert Format and Format-Variable Records 
-#' @param con Bety connection object
+#'
+#' @param con SQL connection to BETYdb
+#' @param format_notes Character string that describes format.
```

Comment:
Do these need the `format_` part? That seems like a given. Also, it's generally good practice to put the required values before the optional ones, and to put a sensible default on optional variables. So, for example, name and mimetype should come before header, skip, and notes. notes can probably default to NULL, skip to 0, and header to TRUE.

Reply:
I don't have an issue with axing the `format_` part. My thought was that the user may need some extra reinforcement that the `names` and `notes` correspond to the formats record and not the variables record. 

And good point! Just fixed the order of the arguments and added these default conditions.